addons/website_sale/views/snippets/s_dynamic_snippet_products.xml

```diff
@@ -20,10 +20,10 @@
                     <we-button data-select-data-attribute="all">All Products</we-button>
                     <we-button data-select-data-attribute="current">Current Category or All</we-button>
                 </we-select>
-                <we-select string="Tag" data-name="product_tag_opt" data-attribute-name="productTagId" data-no-preview="true">
-                    <we-button data-select-data-attribute="all">All Products</we-button>
-                    <we-button data-select-data-attribute="current">Current Tag or All</we-button>
-                </we-select>
+                <t t-set="domain" t-translation="off">["|", ["website_id", "=", false], ["website_id", "=", <t t-out="current_website.id"/>]]</t>
+                <we-many2one string="Tag" data-name="product_tag_opt" data-attribute-name="productTagId" data-limit="15"
+                             data-no-preview="true" data-model="product.tag" t-att-data-domain="domain"
+                             data-select-data-attribute=""/>
```

Comment:
Hello @sdegueldre @qsm-odoo,

I struggled a bit here to find how I could make the selection switchable (an option can be chosen and if the user clicks on it one more time, it empties the selection.)

Is there a way to achieve it ? Where should I find the information about this option if it exists ?

Thanks a lot for your help !

Reply:
@tlibioulle I guess we simply did not provide the option with the widget (we always tried not to transform the editor panel into a backend form view copy, many2one etc are something I'd like to avoid). I have some ideas to actually reuse real backend form views in there... but it's not for tomorrow.

In this case, we could consider adding the option in the `Many2oneUserValueWidget`, it should not be super complicated. However, you have two other solutions:
- Use a we-many2many, could actually make sense for this feature to support multiple tags I think? In that case, you have the possibility to remove records.
- Still use a many2one but add a button to clear by yourself:
```xml
<we-row string="Tag">
    <we-many2one .../>
    <we-button class="fa fa-times" data-select-data-attribute="" data-attribute-name="productTagId"/>
</we-row>
```
Something like that should do the trick. If you want a m2o, maybe the easiest solution would solution 2 then we can create a task on our side to replace your custom button by a widget option if necessary.

Reply:
Thanks a lot for your answer ! :)

I already thought we could use the many2many so I'll give the feedback to PO